### ISSUE TITLE
Undefined constant when building using a locally built viro-react

### ISSUE THREAD

**Issue by cgaydier** [NativeVision/starter-kit#43]
Using node v16.3.0 on windows, in package.json

```
    "@viro-community/react-viro": "file:../ns-viro/viro-community-react-viro-2.23.0.tgz",
    "react": "17.0.2",
    "react-native": "0.65.1"
```

I got an error when trying to run `npm run android`

`TypeError: undefined is not an object (evaluating '_$$_REQUIRE(_dependencyMap[4], "@viro-community/react-viro").ViroConstants.TRACKING_NORMAL') `

The app opens for some seconds and the camera seems to work before it crashes.

Related to #37

**Comment by c4ffein** [comment]
As it was said on the Discord by another user : `change ViroConstants with ViroTrackingStateConstants`
https://viro-community.readme.io/docs/viroconstants

**Comment by c4ffein** [comment]
#39 Already tries to fix this

**Comment by cgaydier** [comment]
Fix thank to #39 and PR #40

**Comment by robertjcolley** [comment]
Fixed in the latest starter kit!
